### PR TITLE
Bug 1373660, added information about the diagnostics tool

### PR DESCRIPTION
**Pull request by ahardin-rh**
https://bugzilla.redhat.com/show_bug.cgi?id=1373660

### CODE REVIEW

**Comment by ahardin-rh** [comment]
@sosiouxme PTAL. Thanks!

**Comment by ahardin-rh** [comment]
After discussion with @adellape, I decided to move this into its own topic.

File: admin_guide/diagnostics_tool.adoc [hunk 1]
```diff
@@ -0,0 +1,51 @@
...
+:icons:
+:experimental:
+:toc: macro
+:toc-title:
+
+toc::[]
+
+== Overview
+
+The `oc adm diagnostics` command launches a diagnostic pod that reports on
```
Comment (sosiouxme) [inline admin_guide/diagnostics_tool.adoc]: launching a diagnostic pod is one thing it does but I wouldn't say that's the overview...

More like:
"... runs a series of checks for error conditions in the host or cluster. Specifically, it:"

**Comment by sosiouxme** [comment]
Otherwise, LGTM!

**Comment by ahardin-rh** [comment]
@sosiouxme Thanks!

**Comment by ahardin-rh** [comment]
[rev_history]
|xref:../admin_guide/diagnostics_tool.adoc #admin-guide-diagnostics-tool[Diagnostics Tool]
|New topic about the `oc adm diagnostics` command.
%

**Pull request merged**